apps/ima/index.js

```diff
@@ -33,9 +33,278 @@ module.exports = {
       behaviours: SaveFormSession,
       fields: ['who-are-you'],
       locals: { showSaveAndExit: true },
-      next: '/confirm', // TO BE UPDATED AS STEPS ARE ADDED
+      next: '/removal-condition',
       backLink: 'current-progress'
     },
+    '/removal-condition': {
+      behaviours: SaveFormSession,
+      fields: ['how-removal-condition-1-applies'],
+      forks: [
+        {
+          target: '/electronic-travel-authorisation',
+          condition: req => {
+            if (req.sessionModel.get('how-removal-condition-1-applies') && req.sessionModel.get('how-removal-condition-1-applies').indexOf('arrived-without-eta') >= 0) {
+              return true;
+            }
+            return false;
+          }
+        },
+        {
+          target: '/entry-clearance',
+          condition: req => {
+            if (req.sessionModel.get('how-removal-condition-1-applies') && req.sessionModel.get('how-removal-condition-1-applies').indexOf('arrived-without-visa') >= 0) {
+              return true;
+            }
+            return false;
+          }
+        },
+        {
+          target: '/travel-ban',
+          condition: req => {
+            if (req.sessionModel.get('how-removal-condition-1-applies') && req.sessionModel.get('how-removal-condition-1-applies').indexOf('travel-ban') >= 0) {
+              return true;
+            }
+            return false;
+          }
+        },
+        {
+          target: '/deportation-order',
+          condition: req => {
+            if (req.sessionModel.get('how-removal-condition-1-applies') && req.sessionModel.get('how-removal-condition-1-applies').indexOf('deportation-order-applies') >= 0) {
+              return true;
+            }
+            return false;
+          }
+        },
+        {
+          target: '/deception',
+          condition: req => {
+            if (req.sessionModel.get('how-removal-condition-1-applies') && req.sessionModel.get('how-removal-condition-1-applies').indexOf('by-deception') >= 0) {
+              return true;
+            }
+            return false;
+          }
+        },
+        {
+          target: '/without-permission',
+          condition: req => {
+            if (req.sessionModel.get('how-removal-condition-1-applies') && req.sessionModel.get('how-removal-condition-1-applies').indexOf('entered-without-permission') >= 0) {
+              return true;
+            }
+            return false;
+          }
+        }
+      ],
+      locals: { showSaveAndExit: true },
+      continueOnEdit: true,
+      next: '/confirm', // TO BE UPDATED AS STEPS ARE ADDED
```

Comment:
Do we need these comments?

Reply:
We've kept them in to make the merging easier when we get to it.

Reply:
Makes sense, IDEs are setup to pick up this approach as it is quite common way to do it https://medium.com/imdoneio/the-imdone-pitch-feedback-welcome-386430accf01